### PR TITLE
Fix CI oversubscription error with Open MPI

### PR DESCRIPTION
**Pull request by matthiasdiener**
Fixes the recent CI failures we have seen (https://github.com/illinois-ceesd/logpyle/actions)

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -49,6 +49,7 @@ jobs:
             runalyzer-gather summary.sqlite log.sqlite
             runalyzer -m summary.sqlite -c 'dbplot(q("select $t_sim, $t_step"))'
 
+            export OMPI_MCA_rmaps_base_oversubscribe=1
```
Comment (inducer) [inline .github/workflows/ci.yaml:52]: You say "on MacOS", but this change doesn't seem limited to MacOS.

Comment (inducer) [inline .github/workflows/ci.yaml:52]: Could you add a comment on what this does and what problem is being addressed by this?

Comment (matthiasdiener) [reply .github/workflows/ci.yaml:52]: Ok, fair. It's an OpenMPI change, that only affects MacOS in CI since Linux uses mpich ;-)

Comment (matthiasdiener) [reply .github/workflows/ci.yaml:53]: Added

**Comment by matthiasdiener** [comment]
(and just to be clear, this fixes the scheduled CI failures that have been happening over the past days)

**Review by inducer — APPROVED** [review]

**Pull request merged**